src/core/Window.cpp

```diff
@@ -430,105 +430,105 @@ void Window::AllKeysUp(void) {
 void Window::OnFullscreenChanged(bool bIsFullscreen) {
```

Comment:
it seems in the header this is named `isNowFullscreen`, i think just `isFullscreen` or `fullscreen` both here and in the header would be better

Reply:
This one was missed.